homeassistant/components/nextdns/strings.json

```diff
@@ -275,7 +275,7 @@
         "name": "Block Twitch"
       },
       "block_twitter": {
-        "name": "Block Twitter"
+        "name": "Block X"
```

Comment:
Honestly, I think it will be misleading. Maybe `Block X (formerly Twitter`?

Reply:
True. Thank you, @bieniu.